ui/src/dashboards/containers/DashboardPage.js

```diff
@@ -223,7 +223,7 @@ class DashboardPage extends Component {
     dashboardActions.addDashboardCellAsync(dashboard)
   }
 
-  handleCloneCell = cell => () => {
+  handleCloneCell = cell => {
```

Comment:
would you mind bringing in `master` to use the various types that now exist for `Cell`s here? and could you make the output explicit (i.e. `: void`)?